### PR TITLE
fix(issuer): adjust wording in confirmation modals if poly already approved

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [x] I linked the issues related to this PR in its description (if any).
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: packages/polymath-issuer/src/actions/ticker.js [hunk 1]
```diff
@@ -33,21 +33,36 @@ export const reserve = (details: Object) => async (
   const { isEmailConfirmed } = getState().pui.account;
   const fee = await SecurityTokenRegistry.registrationFee();
   const feeView = ui.thousandsDelimiter(fee); // $FlowFixMe
+
+  const allowance = await PolyToken.allowance(
+    SecurityTokenRegistry.account,
+    SecurityTokenRegistry.address
+  );
+
+  const isApproved = allowance > fee;
```
Comment (RafaelVidaurre) [inline packages/polymath-issuer/src/actions/ticker.js]: Should be `>=`

File: packages/polymath-issuer/src/actions/token.js [hunk 2]
```diff
@@ -187,26 +187,42 @@ export const issue = (values: Object) => async (
   const fee = await SecurityTokenRegistry.launchFee();
   const feeView = ui.thousandsDelimiter(fee); // $FlowFixMe
 
+  const allowance = await PolyToken.allowance(
+    SecurityTokenRegistry.account,
+    SecurityTokenRegistry.address
+  );
+
+  const isApproved = allowance > fee;
```
Comment (RafaelVidaurre) [inline packages/polymath-issuer/src/actions/token.js]: Should be `>=`

Comment (sajclarke) [reply packages/polymath-issuer/src/actions/token.js]: Fixed in latest commit

File: packages/polymath-issuer/src/actions/ticker.js (hunk 1 above)
Comment (sajclarke) [reply packages/polymath-issuer/src/actions/ticker.js]: Fixed in latest commit

**Pull request merged**